src/unit_test/scripts/nas_outgoing_svcs_ut.py

```diff
@@ -75,7 +75,7 @@ def exec_shell(cmd):
 def test_pre_req_cfg(clear = False, mgmt_ip = '10.11.70.22/8'):
     #config test pre requisite - manangement vrf
     mode = 'OPX'
-    ret = exec_shell('os10-show-version | grep \"OS_NAME.*Enterprise\"')
+    ret = exec_shell('opx-show-version | grep \"OS_NAME.*Enterprise\"')
```

Comment:
Enterprise??

Reply:
It's a UT script that OPX will never use, just ignore it for now until Enterprise moves onto OPX. They may still want to look for that string.